docs/2/technical/microshare-platform-advanced/bundler-configuration.md

```diff
@@ -0,0 +1,579 @@
+        }
+    }
+}
+```
+
+The provided JSON object demonstrates how nested configurations are used to manage different types of alerts and their corresponding incidents. This structure allows the incident bundler robot to match incoming alerts with the appropriate configuration, ensuring that incidents are created and managed according to predefined rules.
+
+#### Structure Overview
+
+- **solutions**: The top-level object that categorizes different types of solutions.
+  - **alert**: A category within solutions that handles various alerts.
+    - **alerts**: Contains specific alert types and their configurations.
+      - **rodent**: Configuration for rodent-related alerts.
+        - **config**: Defines the default configuration for rodent alerts.
+        - **events**: Specific events related to rodent alerts.
+          - **rodent_present**: placeholder for alerts indicating presence of a rodent.
+      - **service**: Configuration for service-related alerts.
+        - **config**: Defines the default config for service alerts.
+        - **events**: Placeholder for additional service-related events.
+    - **config**: Placeholder for additional alert configurations.
+
+  - **clean**: A category within solutions that handles cleaning-related alerts.
+    - **alerts**: Contains specific alert types and their configurations.
+      - **feedback**: Configuration for feedback-related alerts.
+        - **config**: Placeholder for feedback configuration.
+        - **events**: Configurations for different types of events.
+    - **config**: Defines the default incident settings for cleaning alerts.
+
+#### How It Works
+
+When an alert is received, the incident bundler robot matches it with the appropriate nested JSON object based on the type of solution, alert, and format. The robot traverses the nested structure until it finds the right configuration for the specific alert. Once the correct configuration is identified, it overrides the default configuration and uses the new settings to create an incident from the alert. This approach ensures that each type of alert is handled according to its specific requirements, providing a flexible and efficient incident management system.
+
+Example:
+Take the config for the alert type Solutions->clean->alerts->feedback->leak. The configuration looks like:
+
+```
+"config": {
+    "incident": {
+        "priority": "25"
+    }
+}
+```
+
+This configuration is overwritten to the shared default config as so:-
+
+```
+"config": {
+    "incident": {
+        "priority": "25",
+        "title": "New Incident"
+    },
+    "labels": {
+        "complementaryTodo": "Respond to the $<bindings.share.event> event at $<bindings.share.currentLoc.join(\", \")>",
+        "initialTodo": "Respond to the $<bindings.share.event> event at $<bindings.share.currentLoc.join(\", \")>"
+    },
+    "timing": {
+        "globalReminderTime": "P2W",
+        "globalTimeoutTime": "P3W"
+    },
+    "todos": []
+},
+```
+
```

Comment:
this default come from the shared nested config. 
the shared nested config can be seen by opening 
https://app.microshare.io/composer#/views/io.microshare.config/66704123c3cdcf5693d69556
on prod and 
https://dapp.microshare.io/composer#/views/io.microshare.config/66704123c3cdcf5693d69556
on dev

Reply:
In this section, I am trying to show how the config is overwritten so that is why I have only put this part of the config and not the entire config.